en/upgrade.rst

```diff
@@ -37,7 +37,7 @@ Please confirm :ref:`11_2_changes` in the release notes.
 
 **DB migration**
 
-*   Since the DB volume of CUBRID 11.1 and earlier versions are not compatible with the DB volume of CUBRID 11.2, it should be migrated with cubrid unloaddb/loaddb utility. For more detail procedure, see :ref:`migration-from-41`.
+*   Since the DB volume of CUBRID 11.2 and earlier versions are not compatible with the DB volume of CUBRID 11.3, it should be migrated with cubrid unloaddb/loaddb utility. For more detail procedure, see :ref:`migration-from-41`.
```

Comment:
```suggestion
*   Since the DB volume of CUBRID 11.1 and earlier versions are not compatible with the DB volume of CUBRID 11.3, it should be migrated with cubrid unloaddb/loaddb utility. For more detail procedure, see :ref:`migration-from-41`.
```

Reply:
reflected it